docs/graphql/manual/api-reference/schema-metadata-api/actions.rst

```diff
@@ -114,6 +114,10 @@ ActionDefinition
      - true
      - :ref:`WebhookURL <WebhookURL>`
      - The action's webhook URL
+   * - type
+     - false
+     - [ ``mutation`` | ``query`` ]
+     - The type of the action(if ``query`` is chosen, then the kind argument is ignored)
```

Comment:
```suggestion
     - The type of the action (if ``query`` is chosen, then the kind argument is ignored)
```

Reply:
@codingkarthik do we also need to mention this in the kind argument itself?

Reply:
Also, is this enough documentation? Or should we add a section in the main docs too: https://deploy-preview-4309--hasura-docs.netlify.com/graphql/manual/actions/index.html? cc @tirumaraiselvan @rikinsk

Reply:
@marionschleifer 

I have modified the doc of kind and I have explained it there

I have modified the actions doc to include query actions too.

Should there be a separate section on creating a query action (currently we have one for mutation action)?

cc @tirumaraiselvan @rikinsk

Reply:
@tirumaraiselvan @rikinsk I'm not sure how we want to "sell" query actions? Do they have the same importance as mutation actions? Please advise if a separate section is needed on this 🙂